packages/banner/src/constants.js

```diff
@@ -0,0 +1,67 @@
+const constants = themeData => {
+  const base = {
+    colors: {
+      borderColor: {
```

Comment:
Should these colors be from `theme-data`?

Reply:
This isn't being used anymore so I'll delete it.